sys/include/xtimer/tick_conversion.h

```diff
@@ -129,10 +269,13 @@ inline static uint64_t _xtimer_usec_from_ticks64(uint64_t ticks) {
 #else
 #error Unknown hardware timer frequency (XTIMER_HZ), check board.h and/or add an implementation in sys/include/xtimer/tick_conversion.h
 #endif
+/*
+ * End of optimisations
```

Comment:
I would prefer to have this comment on the line with the `#endif` on L275, and related to the if condition. Suggestion:
```c
#endif /* XTIMER_HZ == xxx */
```